src/patternfly/components/DataToolbar/docs/data-toolbar-intro.md

```diff
@@ -0,0 +1,46 @@
+Data toolbar relies on groups (`.pf-c-data-toolbar__group`) and items (`.pf-c-data-toolbar__item`), with default spacer values. Groups and items can be siblings and/or items can be nested within groups. Modifier selectors adjust spacing based on the type of group or item. Each modifier applies a unique CSS variable, therefore, the base spacer value for all elements can be customized and item/groups spacers can be themed individually. The default spacer value for items and groups is set to `--pf-global--spacer--md` or 16px.
```

Comment:
do you think it's worth saying the default spacer value is set to `--pf-c-data-toolbar--spacer--base`, in the context of customizing/theming items/groups? And then you could mention that the value for that is `--pf-global--spacer--md` or 16px. Like

> The default spacer value for items and groups is set to `--pf-c-data-toolbar--spacer--base`, whose value is `--pf-global--spacer--md` or 16px.